content/hardware/ordering.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Choosing and Ordering a MuSHR Car"
```

Comment:
Choosing and Ordering a V4 MuSHR Car. So it is clear we are talking V4

Reply:
I'll make this change once a naming convention has been decided.